escape_test.go

```diff
@@ -35,8 +35,8 @@ func TestEscape(t *testing.T) {
 	if !strings.HasPrefix(s, `digraph "asdf adsf" {
 	"kasdf99 99"->7;
 	"a &lt;&lt; b";
-	"kasdf99 99" [ "<asfd"=1 ];
-	7 [ "<asfd"=1 ];
+	"kasdf99 99" [ color=green ];
```

Comment:
This test testing escaping of string, so maybe you could change green to be something that needs escaping?